### ISSUE TITLE
Concat extras should have 0 score

### ISSUE THREAD

**Issue by jamesclark92** [dropbox/zxcvbn#132]
If I pass first name and last name as extras and then use them combined as the password, that gets score of 1. Intuitively this seems ridiculous. Surely permutations of extras data will be first guesses?

http://plnkr.co/edit/ethkGMI9xeaZ3kOIxhbO?p=preview

**Comment by lowe** [comment]
Good point, agreed that this intuitively seems off -- I'm still thinking about the best way to fix, but to explain the issue: newer versions of the library add guesses for pattern sequence length, and this is being matched as a length-2 pattern sequence (two dictionary words). It's an extreme example because the user_inputs dictionary typically only consists of a few tokens. A workaround in the meantime is to pass the concatenation into the user_inputs dictionary: [first, last, firstlast], but I'll keep this open until a better fix is in place.
